### DISCUSSION TITLE
Body Sensors (heart rate) etc.

### DISCUSSION THREAD

**Discussion by RastalDev** [PranshulGG/WeatherMaster#81] [General]
Is there a reason why the weather app requires   the `android.permission.BODY_SENSORS` permission?

**Comment by PranshulGG** [comment]
> Is there a reason why the weather app requires the `android.permission.BODY_SENSORS` permission?

Yep, I wanted to make the wind direction arrow move with the device's compass, but it didn’t work out. I forgot to remove that permission, so right now it's not being used for anything.
